dotnet/targets/Xamarin.Shared.Sdk.targets

```
@@ -918,6 +919,8 @@
 
 			<_AOTInputDirectory>$(_IntermediateNativeLibraryDir)aot-input/</_AOTInputDirectory>
 			<_AOTOutputDirectory>$(_IntermediateNativeLibraryDir)aot-output/</_AOTOutputDirectory>
+			<_IsDedupEnabled>true</_IsDedupEnabled>
```

```suggestion
			<_IsDedupEnabled Condition="'$(_IsDedupEnabled)' == ''">true</_IsDedupEnabled>
```
Allows overriding the value on a project level.